Impossible-Finance-Agents/New-Pair/package.json

```
@@ -16,18 +20,20 @@
     "disable": "forta-agent disable",
     "enable": "forta-agent enable",
     "keyfile": "forta-agent keyfile",
-    "test": "jest"
+    "test": "jest",
+    "format": "prettier --write \"src/**/*.ts\""
   },
   "dependencies": {
-    "forta-agent": "^0.0.34",
-    "forta-agent-tools": "^1.0.32"
+    "forta-agent": "^0.1.3"  
```

```suggestion
    "forta-agent": "^0.1.6"  
```
Maybe we should match the version to the ones used in the other bots?